### PR TITLE
Improve clipping documentation

### PR DESCRIPTION
**Pull request by mrobinson**
The old documentation was very out of date. The new documentation
is slightly less complete, but should be much more accurate.

This is part of #2666.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2806)

### CODE REVIEW

File: webrender/doc/CLIPPING.md [hunk 1]
```diff
@@ -1,97 +1,53 @@
...
-   fixed.  ([github issue](https://github.com/servo/webrender/issues/1090))
-
-1. It should be possible to specify more than one predefined clip for an item.
-   This is necessary for items that live in a scrolling frame, but are also
-   clipped by a clip that lives outside that frame.
-   ([github issue](https://github.com/servo/webrender/issues/840))
+# Clipping and Positioning in WebRender
+
+Each non-structural WebRender display list item has
+ * A `ClipId` of a positioning node
```
Comment (kvark) [inline webrender/doc/CLIPPING.md]: isn't it a little weird that the positioning node is defined by `ClipId` which has nothing to do with clipping?

Comment (mrobinson) [reply webrender/doc/CLIPPING.md]: It certainly is. I think if we do switch to using two different ids, it makes sense to call one `ClipScrollNodeId` and `ClipChainId` or something like that.

**Pull request merged**